pallets/did/src/did_details.rs

```diff
@@ -576,6 +576,19 @@ impl<T: Config> sp_std::fmt::Debug for DidCreationDetails<T> {
 	}
 }
 
+/// Errors that might occure while deriving the authorization verification key
+/// relationship.
+#[derive(Clone, Debug, Decode, Encode, PartialEq)]
+pub enum RelationshipDeriveError {
+	/// The call is not callable by a did origin.
+	NotCallableByDid,
+
+	/// The parameter of the call where invalid.
```

Comment:
Either singular or plural
```suggestion
	/// The parameter of the call is invalid.
```